index.js

```diff
@@ -3,7 +3,7 @@ const core = require('@actions/core');
 try {
   // Get inputs
   const input1 = parseInt(core.getInput('input1'));
-  const input2 = parseInt(core.getInput('input2'));
```

Comment:
The empty line at the end of the file is unnecessary and can be removed to keep the code clean and concise.